src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -107,6 +104,12 @@ const styles = StyleSheet.create({
   ml8: {
     marginLeft: 8
   },
+  marginNonContinuous: {
+    marginLeft: 4
+  },
+  marginContinuous: {
+    marginLeft: 5
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New margin styles have been added. These are used in the `renderAvatar` function to conditionally apply different margins. This seems like a good approach to handle different conditions for rendering the avatar.



<!-- This is an auto-generated comment by OSS CodeRabbit -->